lib/web_of_science/query_orcid.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module WebOfScience
+  # Use author orcid to find WOS publications for an Author
+  # fetch the wos uids for the given author
+  # e.g. WebOfScience::QueryOrcid.new(author).uids
+  class QueryOrcid
+    def initialize(author, options = {})
+      raise(ArgumentError, 'author must be an Author') unless author.is_a? Author
```

Comment:
Why is this everywhere?

Reply:
yeah, probably overkill, was copypasta.

i removed and left it only in the `QueryAuthor` class, which calls these sub-classes